src/porepy/fracs/fracture_network_2d.py

```diff
@@ -73,20 +87,23 @@ def __init__(self, pts=None, edges=None, domain=None, tol=1e-8):
             self.pts = np.zeros((2, 0))
         else:
             self.pts = pts
+
         if edges is None:
             self.edges = np.zeros((2, 0), dtype=int)
         else:
             self.edges = edges
+
         self.domain = domain
+
         self.tol = tol
 
-        self.tags = {}
-        self.bounding_box_imposed = False
-        self._decomposition = {}
+        self.tags: dict = {}
```

Comment:
If I understood the new doc style correctly, attributes assigned in init should be followed by a docstring.

Reply:
Documentation updated (this line and other class attributes).